bot/resources/tags/under.md

```diff
@@ -0,0 +1,41 @@
+**Use cases of Underscore in Python**
+
+- **Dunders**: Double Underscore (Dunder) methods are special methods defined in a class that are invoked implicitly. For example, `__name__`,`__init__`,`__repr__`, and `__str__`.
+Use the `!dunder-methods` tag to know more about these methods.
+- **Single leading Underscore**:  They're used in front a variable, a function or a method name, which indicates that these objects are meant to be used internally, such as `_name`. However, these objects **can** still be accessed from outside.
+- **Single Trailing Underscores**: They are used to avoid naming conflict while using a variable, which is a reserved keyword. Using a variable name as `class` will produce an error. To avoid this conflict, you can add a trailing underscore as a naming convention i.e. `class_`.
+-  **Double leading underscores**: Double leading underscores are typically used for Name Mangling. Name mangling is a process by which the interpreter changes the attribute name to avoid naming collisions in subclasses. More details regarding Name Mangling can be found [here](https://www.geeksforgeeks.org/name-mangling-in-python/).
```

Comment:
```suggestion
-  **Double Leading Underscores**: Double leading underscores are typically used for "Name Mangling". Name mangling is a process by which the interpreter changes the attribute name to avoid naming collisions in subclasses. More details regarding this topic can be found [here](https://www.geeksforgeeks.org/name-mangling-in-python/).
```